src/components/CodeEditor.vue

```diff
@@ -71,7 +71,7 @@ export default {
       if (this.file && this.file.name) {
         const nameParts = this.file.name.split(".");
         const ext = nameParts[nameParts.length - 1];
-        return languageExts[ext] ? languageExts[ext] === "javascript"? null : languageExts[ext] : "markdown";
+        return languageExts[ext];
```

Comment:
We can use the default fallback language to be markdown in case no extensions available in the name. I can fix it :+1: